lib/wit/main.py

```diff
@@ -108,6 +111,37 @@ def main() -> None:
         raise WitBug(e)
 
 
+def foreach(ws, args):
+    packages = sorted(ws.lock.packages, key=lambda x: str.lower(x.name))
```

Comment:
Rather than sorting, I think it should be left in the order read from the lock file. I thought that in `lock.py` we already lock the packages into an `OrderedDict`, but it looks like we only write them that way. My preference would be to make the ordering deterministic that way instead of via sorting here.

Reply:
Happy to loop in the order they're in the lock file